### DISCUSSION TITLE
How to use Integer or Long range filter in a DataTable Column, similar to Date range?

### DISCUSSION THREAD

**Discussion by escay** [primefaces/community#745] [PrimeFaces] [answered]
In issue https://github.com/primefaces/primefaces/issues/6542 the Data range is implemented and added to the Demo page.
The issue states: 

_Turns out, I made few changes so the constraint can be applied to every comparable objects and not only related to date_

and:

_I saw that and like it. So you could even have it be Integer or anything else than is Comparable._

But how should I implement this in the xhtml in the filter facet?
The demo:
https://www.primefaces.org/showcase/ui/data/datatable/filter.xhtml

uses datePicker with selectionMode attribute:
```
            <p:column field="date" headerText="Join Date" filterMatchMode="between">
                <f:facet name="filter">
                    <p:datePicker selectionMode="range" onchange="PF('customersTable2').filter()" styleClass="ui-custom-filter"/>
                </f:facet>
            </p:column>
```
But is there a input text solution for Integer or Long columns?

**Comment by melloware** [comment, marked as answer]
You can read about it here: https://github.com/primefaces/primefaces/issues/8452  I am pretty sure this is what you are asking?

**Reply by escay** [reply]
Thank you, that is the feature I am looking for. (I would prefer textbox like used for the date range over  slider)

**Reply by melloware** [reply]
@escay you should go comment on that ticket and be clear with what you want so there is even more info in the ticket.
